templates/pyproject.toml

```diff
@@ -0,0 +1,205 @@
+[build-system]
+# Defined by PEP 518
+requires = [
+# Some packages will require extra additions to this list. E.g "Cython" or "Numpy"
+  "setuptools>=61",
+  "setuptools_scm[toml]>=7",
+  "wheel"
+]
+# Defined by PEP 517
+build-backend = "setuptools.build_meta"
+
+[project]
+authors = [
+    {name = "<PACKAGE_NAME> Contributors", email = "scitools.pub@gmail.com"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Science/Research",
+    "License :: OSI Approved :: BSD License",
+    "Operating System :: THIS VARIES BY REPOSITORY",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: SUPPORTED PYTHON VERSIONS",
+    # e.g "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: Implementation :: CPython",
+    "Topic :: Scientific/Engineering",
+    "Topic :: Scientific/Engineering :: Atmospheric Science",
+    "Topic :: Scientific/Engineering :: VARIES BY REPOSITORY",
+]
+dynamic = [
+    "dependencies",
+    "readme",
+    "version",
+]
+description = "A powerful, format-agnostic, community-driven Python package for analysing and visualising Earth science data"
+keywords = [
+    "<PACKAGE_KEYWORDS>",
+]
+license = {text = "BSD-3-Clause"}
+name = "scitools-<PACKAGE_NAME>" #e.g "scitools-iris"
+requires-python = "<MIN_PYTHON_VERSION>"
+
+[project.urls]
+Code = "https://github.com/SciTools/<PACKAGE_NAME>"
+Discussions = "https://github.com/SciTools/<PACKAGE_NAME>/discussions" # If discussions are enabled.
+Documentation = "https://scitools-iris.readthedocs.io/" # If you have a documentation site.
+Issues = "https://github.com/SciTools/<PACKAGE_NAME>/issues" # If issues are enabled.
+
+[tool.check-manifest]
+# Ignores here will vary by repository.
+ignore = []
```

Comment:
This will always include the version file specified in the `[tool.setuptools_scm]` section.